src/FEValues/CellValues.jl

```diff
@@ -0,0 +1,72 @@
+include("GeometryValues.jl")
+include("FunctionValues.jl")
+
+function default_geometric_interpolation(::Interpolation{shape}) where {dim, shape <: AbstractRefShape{dim}}
+    return VectorizedInterpolation{dim}(Lagrange{shape, 1}())
+end
+
+struct CellValues{IP, N_t, dNdx_t, dNdξ_t, T, dMdξ_t, QR, GIP} <: AbstractCellValues
+    geo_values::GeometryValues{dMdξ_t, GIP, T}
+    fun_values::FunctionValues{IP, N_t, dNdx_t, dNdξ_t}
```

Comment:
What is the rationale behind using concrete types here?

Reply:
To keep the parameterization of `CellValues`. But I think that should not be required, since its not required by `AbstractCellValues`. Fredrik and I discussed adding a note in the documentation before releasing 1.0 that the type-parameterization of `CellValues` and `FaceValues` might change. 

I think we should rather provide functions for accessing those types.